### ISSUE TITLE
Test suite failure on GHC 7.10

### ISSUE THREAD

**Issue by snoyberg** [ekmett/hyperloglog#10]
```
Test suite doctests: RUNNING...
### Failure in src/Data/HyperLogLog/Type.hs:89: expression `mempty :: HyperLogLog $(3)'
expected: HyperLogLog {runHyperLogLog = fromList [0,0,0,0,0,0,0,0]}
 but got:
          <interactive>:44:25:
              Illegal type: ‘3’ Perhaps you intended to use DataKinds
              In the splice: $(3)
Examples: 7  Tried: 6  Errors: 0  Failures: 1
Test suite doctests: FAIL
Test suite logged to: /home/vagrant/haskell/stackage/logs/nightly/hyperloglog-0.3.1/test-run.out
```

**Comment by jimstutt** [comment]
I guess this is on your agenda but it's a stopper for working on hlearn as that depends on subhask which shows the same failure.

**Comment by mikeizbicki** [comment]
@jimstutt There's no need to run the test suite for hyperloglog when installing subhask.  `cabal install subhask` works fine for me and generates no errors wrt hyperloglog.

**Comment by jimstutt** [comment]
Thanks for the pointer. I had successfully packaged subhask as a nix pkg
with cabal2nix. I commented out the test sections of subhask.cabal but that
didn't work; I'll have to dig in to preventing a local copy of hyperloglog
from running the test suite.

**Comment by glguy** [comment]
Build fixed by 038787841515670e9ba3ce8be97474d1cbd561eb
